docs/use_case/postv1.md

```diff
@@ -0,0 +1,61 @@
+This initial version of the demo #post app will allow you to create and visualise your own post(s). It will also showcase examples of the ONEM's python SDK usage.
+
+So let's activate our virtualenv and navigate to the **postv1** folder. Make sure that:
+
+ - the database is created (even if we will store the posts in-memory).
+ - local django server is started.
+ - the server is exposed publicly.
+ - the application is registered.
+All of the above steps are explained in the [run locally](/getting_started/python_run_local/) section of the docs.
+
+Now we can access the app from the test client. We should get the below menu:
```

Comment:
Tell them to send #<app name> and mention how to obtain the app name.  I'm trying here to be explicit about all the commands needs, apart from the pre-requisites.  That way, this POST section could be separated out into a separate article later if needed.